Library/Formula/fontconfig.rb

```diff
@@ -1,7 +1,8 @@
 class Fontconfig < Formula
   desc "XML-based font configuration API for X Windows"
   homepage "http://fontconfig.org/"
-  url "http://fontconfig.org/release/fontconfig-2.11.1.tar.bz2"
+  url "https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.11.1.tar.bz2"
+  mirror "http://fontconfig.org/release/fontconfig-2.11.1.tar.bz2"
```

Comment:
Thanks @DomT4  - I'm glad you confirmed my suspicions. The homepage was still existing yesterday, perhaps we caught them in a state of transition. Either way I've fixed the formula as per your advice.